feats/redis.py

```diff
@@ -35,9 +37,20 @@ class FeatureStream:
     a RedisClient instance.
     """
     def __init__(self, redis, key):
-        self.key = f"feature:{key}"
+        self.key = self._get_key(key)
         self._redis = redis
 
+    def _get_key(self, key) -> str:
+        """
+        Builds the Redis stream key. If the `FEATS_ENV` environment variable is
+        set, it will be used as the prefix for the feature key.
+        """
+        env = os.getenv('FEATS_ENV')
```

Comment:
What if we had this be an input to the RedisClient object instead? That way we aren't dependent on a global variable like the environment.

Reply:
I could just be misunderstanding the way feats interacts with the rest of the world, but would that mean we'd need to push different code to each environment? I.e. with `RedisClient(xyz, prefix='staging=23')` for staging, `RedisClient(xyz, prefix=None)` for prod?

Reply:
Different values would need to be passed, but the settings file knows which environment it is being deployed to, and can combine that with any environment variables we need. 

For example, our memcache setting grabs the key prefix from the environment and passes that through to the memcache client -- the client itself has no knowledge that it came from os.environ.

Reply:
👍  I like it (completely forgot there was a settings file 😅 )